reference-architecture/azure-ansible/node.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#yum -y update
+yum -y install wget git net-tools bind-utils iptables-services bridge-utils bash-completion docker
```

Comment:
I believe this is only in regards to the host performing the installation. If you check out the aws plays we don't actually install those packages.